ext/opcache/jit/zend_jit_ir.c

```diff
@@ -14376,8 +14376,7 @@ static int zend_jit_fetch_obj(zend_jit_ctx         *jit,
 				if (ce && ce->ce_flags & ZEND_ACC_IMMUTABLE) {
 					ref = ir_CONST_ADDR(prop_info);
 				} else {
-					int prop_info_offset =
-						(((prop_info->offset - (sizeof(zend_object) - sizeof(zval))) / sizeof(zval)) * sizeof(void*));
+					int prop_info_offset = Z_PROP_TABLE_OFFSET(prop_info) * sizeof(void*);
```

Comment:
Actually, I think the changes in this file are not needed. Currently, `prop_info` will not be available when the property contains hook, and if they don't we're already working with the parent property.